src/main/java/org/sonarsource/sonarlint/ls/SonarLintExtendedLanguageClient.java

```diff
@@ -75,19 +75,19 @@ class ShowRuleDescriptionParams {
     @Expose
     private final String htmlDescription;
     @Expose
-    private final RuleType type;
+    private final String type;
```

Comment:
Why change back? Doesn't it work this way?

Reply:
[This](https://sonarsource.slack.com/archives/C03M0AJ3BKR/p1661508347180749) is the related slack thread. For client-side the format should remain the same because some functionalities depend on it being a String.